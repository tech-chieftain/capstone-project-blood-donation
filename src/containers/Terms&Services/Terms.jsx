import React from 'react';
import { useTranslation } from 'react-i18next';
import Paragraph from '../../components/Paragraph';

function Terms() {
  const { t } = useTranslation();
  const styleClass = {
    HeaderStyle: `text-primary  md:text-lg lg:text-xl font-medium  pb-5 pt-5`,
    TextStyle: `sm:text-sm md:text-base lg:text-lg leading-8 `,
  };

  return (
    <div className="p-10 ">
      <Paragraph
        headerText={t('Terms.TermsAndServices')}
        paragraphText={t('Terms.TermsText')}
        headerFlag
        paragraphFlag
      />
      <h1 className={styleClass.HeaderStyle}>{t('Terms.TermsOfUse')}</h1>
      <ol className="list-decimal">
        <li className={styleClass.TextStyle}>{t('Terms.firstTerm')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.secondTerm')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.thirdTerm')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.forthTerm')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.fifthTerm')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.sixthTerm')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.seventhTerm')}</li>
      </ol>
      <h1 className={styleClass.HeaderStyle}> {t('Terms.Services')}</h1>
      <ol className="list-decimal">
        <li className={styleClass.TextStyle}>{t('Terms.firstService')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.secondService')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.thirdService')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.forthService')}</li>
        <li className={styleClass.TextStyle}>{t('Terms.fifthService')}</li>
      </ol>
    </div>
  );
}

export default Terms;
