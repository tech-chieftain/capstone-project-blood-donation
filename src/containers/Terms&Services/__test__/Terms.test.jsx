import React from 'react';
import renderer from 'react-test-renderer';
import Terms from '../Terms';

// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer.create(<Terms />).toJSON();
  expect(tree).toMatchSnapshot();
});
