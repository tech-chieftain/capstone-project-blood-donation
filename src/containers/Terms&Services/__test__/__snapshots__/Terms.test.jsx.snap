// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="p-10 "
>
  <div>
    <h1
      className="text-primary-100  sm:text-xl md:text-2xl lg:text-3xl pt-1 font-semibold  text-center"
    >
      Terms.TermsAndServices
    </h1>
    <p
      className=" pt-3 pb-5  sm:text-sm md:text-base lg:text-lg  text-center "
    >
      Terms.TermsText
    </p>
  </div>
  <h1
    className="text-primary  md:text-lg lg:text-xl font-medium  pb-5 pt-5"
  >
    Terms.TermsOfUse
  </h1>
  <ol
    className="list-decimal"
  >
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.firstTerm
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.secondTerm
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.thirdTerm
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.forthTerm
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.fifthTerm
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.sixthTerm
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.seventhTerm
    </li>
  </ol>
  <h1
    className="text-primary  md:text-lg lg:text-xl font-medium  pb-5 pt-5"
  >
     
    Terms.Services
  </h1>
  <ol
    className="list-decimal"
  >
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.firstService
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.secondService
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.thirdService
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.forthService
    </li>
    <li
      className="sm:text-sm md:text-base lg:text-lg leading-8 "
    >
      Terms.fifthService
    </li>
  </ol>
</div>
`;
